### DISCUSSION TITLE
Fall guys not working with the latest proton

### DISCUSSION THREAD

**Discussion by jrthe420tokerr** [community/community#32838] [Other Feature Feedback, Questions, & Ideas]
Hello everyone. had to reimage my deck. could not get the proton ge v 32 that i was using to play fall guys. no other version seems to be working with fall guys. also just saw i cant get the .32 version anymore. is there an other way to get fall guys running. my kids are driving me nuts. thats all the like to play. someone please help. the game loads all the way but when i try to join a game tells me error could not connect to their servers

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, though I'm not sure which, there are a bunch of repositories with "Fall Guys" in their names. Anyway, in that case you should open an issue *in the repository for that software* so the developers see it. :slightly_smiling_face:
